tests/scripts/all.sh

```diff
@@ -2386,8 +2386,7 @@ component_test_psa_crypto_config_accel_ecjpake_use_psa () {
 
     # Build the main library
     loc_accel_flags="$loc_accel_flags $( echo "$loc_accel_list" | sed 's/[^ ]* */-DMBEDTLS_PSA_ACCEL_&/g' )"
-    #make CFLAGS="$ASAN_CFLAGS -O -Werror -I../tests/include -I../tests -I../../tests -DPSA_CRYPTO_DRIVER_TEST -DMBEDTLS_TEST_LIBTESTDRIVER1 $loc_accel_flags" LDFLAGS="-ltestdriver1 $ASAN_CFLAGS"
-    make CFLAGS="$ASAN_CFLAGS -O0 -g -Werror -I../tests/include -I../tests -I../../tests -DPSA_CRYPTO_DRIVER_TEST -DMBEDTLS_TEST_LIBTESTDRIVER1 $loc_accel_flags" LDFLAGS="-ltestdriver1 $ASAN_CFLAGS"
+    make CFLAGS="$ASAN_CFLAGS -O -Werror -I../tests/include -I../tests -I../../tests -DPSA_CRYPTO_DRIVER_TEST -DMBEDTLS_TEST_LIBTESTDRIVER1 $loc_accel_flags" LDFLAGS="-ltestdriver1 $ASAN_CFLAGS"
```

Comment:
For the record: `ASAN_CFLAGS` includes `-O2` and I don't see a reason to change that to a lower optimization level here. It also includes `-Werror` btw, so we don't need to use it here either.

Not worth changing, especially as I'm hoping to merge the three components for ECDH, ECDSA, and ECJPAKE when wrapping up this EPIC. Just mentioning FYI.

Reply:
Sure, I just followed calls from other components that are building the library. Improvement area: we could define this long command somewhere to reuse it, there are currently 5 places that use it.

Reply:
Yes, sorry for those changes, they are all connected to my debug sessions: being able to go through each line of the code without optimizations makes life easier for me. I must pay more attention when I add things for the commit